### PR TITLE
#166462564: User should see an improved landing page

### PR DESCRIPTION
**Pull request by bisonlou**
**What does this PR do?**
• Creates a landing page Displaying Most Popular, Most Liked, and Most Recent articles.
• Adds Home and Articles Navigation Buttons.
• Adds Moto to Navigation Bar.
• Adds a loader.

**Description of Task to be completed?**
• Users should be directed to the landing page as the first page

**How should this be manually tested?**
[Follow the link](https://ah-frontend-dojo-pr-22.herokuapp.com/)

**Any background context you want to provide?**
The app requires a landing page.

**What are the relevant pivotal tracker stories?**
[166462564](https://www.pivotaltracker.com/n/projects/2326197)

**Screenshots (if appropriate)**

### CODE REVIEW

File: src/store/reducers/articles/Articles.test.js [hunk 1]
```diff
@@ -1,43 +1,21 @@
+import { sampleArticle } from 'utils';
 import articles from './index';
 import {
   GET_ARTICLES,
+  GET_ARTICLES_START,
+  GET_ARTICLES_ERROR,
 } from '../../actions/types';
```
Comment (kalsmic) [inline src/store/reducers/articles/Articles.test.js]: @bisonlou, kindly change this import to 
```
import {
  GET_ARTICLES,	  GET_ARTICLES,
  GET_ARTICLES_START,
  GET_ARTICLES_ERROR,
} from 'store/actions/types';	
```

**Review by kalsmic — CHANGES_REQUESTED** [review]
@bisonlou, Kindly look into this

**Review by llwasampijja — APPROVED** [review]
Nice work @bisonlou The page displays well, the code is organized and commented well.

**Review by kalsmic — APPROVED** [review]

**Review by JamesMudidi — APPROVED** [review]
@bisonlou Well done. I have just tested your work and it works as expected.
I have loved the comments on the code aswell

**Review by nadralia — APPROVED** [review]

**Review by ZackAtama — APPROVED** [review]
Great work @bisonlou

**Pull request merged**